### DISCUSSION TITLE
The scope and limitations of the future blueprint feature

### DISCUSSION THREAD

**Discussion by DBT85** [voxeltycoon/issues#722] [General]
Just wanted to see what people felt about the limitations or lack thereof the blueprint function should have.

The only other game I've played a lot of with such a feature was Rise of Industry where a purposeful limit was placed on how much could be saved to a blueprint to prevent someone just plonking down a complete industrial area with everything already worked out perfectly like 5 farms, a warehouse, 5 factories making x, 3 factories making y etc. 

This is the route I assume VT would go down, but I wonder what others think. Should you just be able to save or download a perfect block of unlimited size to immediately have the perfect ratios and layout to plonk down? Should it be limited to maybe 1 building and attached conveyors? Still allowing less time consuming patterning but forcing at least a little effort on the player?

**Comment by victorleblais** [comment]
I'm mostly familiar with Factorio's blueprint, which are basically restriction-less.

I personally do not use any blueprints made by other players, but I don't have anything against anyone that does so. I think it's better to have more freedom: let people play the way they want I guess. Everyone has fun its own way, and I totally understand why some people might now want to mess with certain aspects of the game, if that's what they want.
